drake/util/drakeGeometryUtil.h

```diff
@@ -295,4 +301,4 @@ DLLEXPORT void quat2expmapSequence(const Eigen::Ref<const Eigen::Matrix<double,4
 DLLEXPORT GradientVar<double,3,1> closestExpmap(const Eigen::Ref<const Eigen::Vector3d> &expmap1, const Eigen::Ref<const Eigen::Vector3d> &expmap2, int gradient_order);
 
 
-#endif
+#endif
```

Comment:
It's good practice to add a newline at the end of the file because it's a *nix convention and some *nix tools don't work as expected without one (e.g. cat will put the prompt at the end of the last line, so if that last line is not empty...). Also the C standard officially says you need that newline (but C++11 will treat a file without one as if there were one). It's not really a big deal.